src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandler.java

```diff
@@ -59,8 +63,33 @@ public boolean canHandle(
             throw new IllegalStateException("Cannot handle callback");
         }
 
+        AsylumCase asylumCase =
+            callback
+                .getCaseDetails()
+                .getCaseData();
+
+        if (callback.getEvent() == Event.SEND_DIRECTION
+            && getlatestNonStandardRespondentDirection(asylumCase).isEmpty()) {
+
+            return new PreSubmitCallbackResponse<>(asylumCase);
+        }
+
         AsylumCase asylumCaseWithHomeOfficeData = homeOfficeApi.call(callback);
 
         return new PreSubmitCallbackResponse<>(asylumCaseWithHomeOfficeData);
     }
+
+    protected Optional<Direction> getlatestNonStandardRespondentDirection(AsylumCase asylumCase) {
```

Comment:
could you please fix the name to adhere to standard convention?

Reply:
Changed the name to standard